plugins/autocertkit

```diff
@@ -1279,10 +1279,20 @@ def reset_arp(session, args):
         vm_m_ip = wait_for_ip(session, vm_ref, 'eth0')
 
         # Global ARP Ignore
-        cmd = "%s net.ipv4.conf.all.arp_ignore=1" % SYSCTL
+        cmd = "%s net.ipv4.conf.all.arp_ignore=0" % SYSCTL
         call = [SSH, vm_m_ip, cmd]
         make_local_call(call)
 
+        '''
+        # Revert arp_ignore to default 0, otherwise DLVM (CentOS 7.2) no interface will reply arp query.
```

Comment:
If this is no longer required, doesn't that mean we can remove the command altogether? Assuming 0 is the default?

Reply:
Yes. 0 is default. But just keep code here without harm. I am not sure if the situation will change when update DLVM in next time, e.g. CentOS8